llvm/test/CodeGen/AMDGPU/commute-op-sel.mir

```diff
@@ -0,0 +1,17 @@
+# RUN: llc -mtriple=amdgcn -mcpu=gfx1030 -run-pass=machine-cse -verify-machineinstrs %s -o - 2>&1 | FileCheck --check-prefix=GCN %s
+
+# GCN-LABEL: name: test_machine_cse_op_sel
+# GCN: %2:vgpr_32 = V_ADD_NC_U16_e64 0, %0, 0, %1, 1, 0, implicit $mode, implicit $exec
+# GCN: %3:vgpr_32 = V_ADD_NC_U16_e64 0, %1, 0, %0, 1, 0, implicit $mode, implicit $exec
```

Comment:
No commute happened, and the modifiers aren't set?

Reply:
It seems we do not define VOP3 instructions as commutable. Even if both op_sel are 0, the two V_ADD_NC_U16_e64 are not merged.

Reply:
That's a bug that should be fixed

Reply:
Can it be done with separate PR? This PR is for SDWA commute issue which causes rocFFT to fail. Whereas VOP3 not commute is missed performance opportunity. The current issue is kind of urgent.

Reply:
OK I just noticed you have approved this PR. I will commit it first and will look into VOP3 commuting opportunity later.

Reply:
open an issue to track this https://github.com/llvm/llvm-project/issues/111205